html/cross-origin-opener-policy/access-reporting/openee-accessed_openee-coop.https.html

```diff
@@ -75,10 +75,17 @@
     report_2 = JSON.parse(report_2);
     assert_equals(report_2.length, 1);
     assert_equals(report_2[0].type, "coop");
-    assert_equals(report_2[0].body["violation-type"], "access");
+    assert_equals(report_2[0].url, openee_url);
+    assert_equals(report_2[0].body["violation-type"], "access-to-coop-page");
     assert_equals(report_2[0].body["disposition"], "enforce");
     assert_equals(report_2[0].body["effective-policy"], "same-origin-plus-coep");
-  }, `${test}`);
+    assert_equals(report_2[0].body["property"], property);
+    assert_equals(report_2[0].body["source-file"], undefined);
+    assert_equals(report_2[0].body["lineno"], undefined);
+    assert_equals(report_2[0].body["colno"], undefined);
+    // TODO(arthursonzogni): Add check for report > body > blocked-window-url
+
+  }, `${type} ${test}`);
```

Comment:
```suggestion
  }, `${type} ${property}`);
```

cc @ArthurSonzogni looks like a typo here